Build failure on some linux build servers

On some of my build servers (debian 12, openSuse 15.6), I am getting the following error for last two days. I am not sure what changed! 

```
 Compiling magika v0.1.0-rc.2 (https://github.com/google/magika#906afc82)
error[E0432]: unresolved import `ort::GraphOptimizationLevel`
  --> /home/dilawar/.cargo/git/checkouts/magika-0dc2ca1a149e7051/906afc8/rust/lib/src/builder.rs:15:5
   |
15 | use ort::GraphOptimizationLevel;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `GraphOptimizationLevel` in the root 
   |
help: consider importing this enum instead
   |
15 | use ort::session::builder::GraphOptimizationLevel;
   |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0433]: failed to resolve: could not find `Session` in `ort`
  --> /home/dilawar/.cargo/git/checkouts/magika-0dc2ca1a149e7051/906afc8/rust/lib/src/builder.rs:55:32
   |
55 |         let mut session = ort::Session::builder()?;
   |                                ^^^^^^^ could not find `Session` in `ort`
   |
help: consider importing one of these structs
   |                                                           
```

Either of the following causes the same error. 

```
  # magika = { git = "https://github.com/google/magika", features = [ "_test", "serde" ]}                                                             
  magika = { version = "0.1.0-rc.2", features = [ "_test", "serde" ]}                                                                                 
```

--- a/rust/cli/CHANGELOG.md
+++ b/rust/cli/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.0-rc.3-dev
+
+### Patch
+
+- Update dependencies
+
 ## 0.1.0-rc.2
 
 ### Minor

--- a/rust/cli/Cargo.toml
+++ b/rust/cli/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "magika-cli"
-version = "0.1.0-rc.2"
+version = "0.1.0-rc.3-dev"
 authors = ["Magika Developers <magika-dev@google.com>"]
 license = "Apache-2.0"
 edition = "2021"
@@ -20,9 +20,9 @@ anyhow = "1.0.86"
 async-channel = "2.3.1"
 clap = { version = "4.5.9", features = ["cargo", "derive", "string"] }
 colored = "2.1.0"
-magika = { version = "=0.1.0-rc.2", path = "../lib", features = ["serde"] }
+magika = { version = "=0.1.0-rc.3-dev", path = "../lib", features = ["serde"] }
 num_cpus = "1.16.0"
-ort = "2.0.0-rc.8"
+ort = "=2.0.0-rc.9"
 serde = { version = "1.0.204", features = ["derive"] }
 serde_json = "1.0.120"
 tokio = { version = "1.38.1", features = ["full"] }

--- a/rust/cli/src/main.rs
+++ b/rust/cli/src/main.rs
@@ -23,7 +23,7 @@ use anyhow::{bail, ensure, Result};
 use clap::{Args, Parser};
 use colored::{ColoredString, Colorize};
 use magika::{ContentType, Features, FeaturesOrRuled, FileType, RuledType, Session, TypeInfo};
-use ort::GraphOptimizationLevel;
+use ort::session::builder::GraphOptimizationLevel;
 use serde::Serialize;
 use tokio::fs::File;
 use tokio::io::AsyncReadExt;

--- a/rust/lib/CHANGELOG.md
+++ b/rust/lib/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.0-rc.3-dev
+
+### Patch
+
+- Update dependencies
+
 ## 0.1.0-rc.2
 
 ### Patch

--- a/rust/lib/Cargo.toml
+++ b/rust/lib/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "magika"
-version = "0.1.0-rc.2"
+version = "0.1.0-rc.3-dev"
 authors = ["Magika Developers <magika-dev@google.com>"]
 license = "Apache-2.0"
 edition = "2021"
@@ -27,10 +27,9 @@ thiserror = "1.0.63"
 tokio = { version = "1.38.1", features = ["fs", "io-util"] }
 
 [dependencies.ort]
-version = "2.0.0-rc.8"
+version = "=2.0.0-rc.9"
 default-features = false
-# TODO: Remove "half" when rc.9 is released.
-features = ["ndarray", "half"]
+features = ["ndarray"]
 
 [dev-dependencies]
 data-encoding = "2.6.0"

--- a/rust/lib/src/builder.rs
+++ b/rust/lib/src/builder.rs
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use ort::GraphOptimizationLevel;
+use ort::session::builder::GraphOptimizationLevel;
 
 use crate::{Result, Session};
 
@@ -52,7 +52,7 @@ impl Builder {
 
     /// Consumes the builder to create a Magika session.
     pub fn build(self) -> Result<Session> {
-        let mut session = ort::Session::builder()?;
+        let mut session = ort::session::Session::builder()?;
         let Builder { inter_threads, intra_threads, optimization_level, parallel_execution } = self;
         if let Some(num_threads) = inter_threads {
             session = session.with_inter_threads(num_threads)?;

--- a/rust/lib/src/future.rs
+++ b/rust/lib/src/future.rs
@@ -38,8 +38,8 @@ pub(crate) trait Env {
     async fn symlink_metadata(path: &Path) -> Result<Metadata>;
     async fn open(path: &Path) -> Result<Self::File>;
     async fn ort_session_run(
-        session: &ort::Session, input: Array2<i32>,
-    ) -> Result<ort::SessionOutputs>;
+        session: &ort::session::Session, input: Array2<i32>,
+    ) -> Result<ort::session::SessionOutputs>;
 }
 
 pub(crate) enum SyncEnv {}
@@ -55,8 +55,8 @@ impl Env for SyncEnv {
     }
 
     async fn ort_session_run(
-        session: &ort::Session, input: Array2<i32>,
-    ) -> Result<ort::SessionOutputs> {
+        session: &ort::session::Session, input: Array2<i32>,
+    ) -> Result<ort::session::SessionOutputs> {
         Ok(session.run(ort::inputs!("bytes" => input)?)?)
     }
 }
@@ -74,8 +74,8 @@ impl Env for AsyncEnv {
     }
 
     async fn ort_session_run(
-        session: &ort::Session, input: Array2<i32>,
-    ) -> Result<ort::SessionOutputs> {
+        session: &ort::session::Session, input: Array2<i32>,
+    ) -> Result<ort::session::SessionOutputs> {
         Ok(session.run_async(ort::inputs!("bytes" => input)?)?.await?)
     }
 }

--- a/rust/lib/src/session.rs
+++ b/rust/lib/src/session.rs
@@ -23,7 +23,7 @@ use crate::{AsyncInput, Builder, Features, FeaturesOrRuled, FileType, Result, Sy
 /// A Magika session to identify files.
 #[derive(Debug)]
 pub struct Session {
-    pub(crate) session: ort::Session,
+    pub(crate) session: ort::session::Session,
 }
 
 impl Session {

--- a/rust/onnx/build.sh
+++ b/rust/onnx/build.sh
@@ -33,8 +33,8 @@ else
   git clone --recursive https://github.com/Microsoft/onnxruntime.git runtime
   cd runtime
 
-  info "Checkout v1.19.2 because that's what ort v2.0.0-rc.8 supports."
-  git checkout v1.19.2
+  info "Checkout v1.20.0 because that's what ort v2.0.0-rc.9 supports."
+  git checkout v1.20.0
 
   info "Build the static libraries."
   x ./build.sh --config=Release --parallel $ONNX_RUNTIME_BUILD_FLAGS

